lib/Cargo.toml

```diff
@@ -51,18 +51,13 @@ rustls = [
     "reqwest?/rustls-tls",
     "tokio-tungstenite?/rustls-tls-webpki-roots",
 ]
-ml = ["surrealml-core", "ndarray"]
-jwks = ["dep:reqwest"]
-arbitrary = [
-    "dep:arbitrary",
-    "dep:regex-syntax",
-    "rust_decimal/rust-fuzz",
-    "geo-types/arbitrary",
-    "uuid/arbitrary",
-]
-experimental-parser = ["dep:phf", "dep:unicase"]
+ml = ["surrealdb-core1/ml", "surrealdb-core2/ml"]
+jwks = ["surrealdb-core2/jwks", "sql2"]
+arbitrary = ["surrealdb-core2/arbitrary", "sql2"]
+sql2 = []
```

Comment:
Is `sql2` the right name? It may be confused with `parser2`, as the new parser involves changes in our SQL dialect.

So, if I understand well, it stands for `SurrealDB2` Shall we use `surdb2`, `surreal2`, `surrealdb2` ?

Reply:
The name `sql` was motivated by the module name. The core crate only publicly exports the `sql` module. So `sql` here is referring to SQL types from the `sql` module. I'm open to suggestions though. `parser2` is specifically the parser functionality, so it's a subset of `sql`.

> So, if I understand well, it stands for `SurrealDB2`. Shall we use `surdb2`, `surreal2`, `surrealdb2`?

The Rust SDK is the public API of SurrealDB with the crate name being `surrealdb` because that's what users will use for both embedded and remote use-cases. Wouldn't these names be confused with the SurrealDB crate?

Reply:
ok, so my assumption was wrong, sql2 really means sql v2. Find by me!
https://crates.io/crates/surrealdb-sql